sdk/src/actor.rs

```diff
@@ -62,6 +62,13 @@ pub fn create_actor(actor_id: ActorID, code_cid: &Cid) -> SyscallResult<()> {
     unsafe { sys::actor::create_actor(actor_id, cid.as_ptr()) }
 }
 
+/// Installs or ensures an actor code CID is valid and loaded.
+#[cfg(feature = "m2-native")]
+pub fn install_actor(code_cid: &Cid) -> SyscallResult<()> {
+    let cid = code_cid.to_bytes();
+    unsafe { sys::actor::install_actor(cid.as_ptr()) }
+}
```

Comment:
Hm. This is going to be a privileged API, I'm not sure if we should even bother adding it to the SDK.

But let's just add a comment noting that it's privileged for now.

Reply:
In the future, many of these methods will move directly into the responsible actors.

Reply:
done.